doc/source/building/working_with_images/vagrant_setup.rst

```diff
@@ -97,15 +117,20 @@ the following changes are required:
       #--------------------------------------
       suseInsertService sshd
 
-   Also make sure to setup **UseDNS=no** in :file:`etc/ssh/sshd_config`
-   This can be done by an overlay file or clever patching of
-   the file in the above mentioned :file:`config.sh` file.
+   Also make sure to setup **UseDNS=no** in :file:`/etc/ssh/sshd_config`
```

Comment:
"to setup ..." as a phrase does not make sense to me. Either you **set** a value to a key, or you add a line. Here are the alternatives:

* Also make sure to set ...
* Also make sure to add the line ...